metadata-ingestion/src/datahub/ingestion/transformer/transform_registry.py

```diff
@@ -0,0 +1,4 @@
+from datahub.ingestion.api.registry import Registry
+from datahub.ingestion.api.transform import Transformer
+
+transform_registry = Registry[Transformer]()
```

Comment:
This isn't necessary for this the initial PR, but I think it will eventually make sense to build a entrypoint-based registry system like sources/sinks have.

Reply:
I'm not sure I understand. Do you think we should remove this in this PR?
The reason I added this is to get for free the dynamic class loading capability of the registry.

Reply:
This is totally fine for this PR! What I meant is that eventually (in a later PR) we should start using the automatic loading + aliasing features of the registry as well.

For example, we'd call `transform_registry.load("datahub.ingestion.transform.plugins")`, and then adding items to setup.py would get automatically registered and have easy-to-remember aliases instead of full class paths. The source_registry.py is a good example of how this is done.